search-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import { RouteNames } from '@/enums'
+import { mount } from '@vue/test-utils'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { BcrsBreadcrumb } from '@/bcrs-common-components'
+import { SearchDashboardBreadcrumb, SearchSIGNINBreadcrumb } from '@/bcrs-common-components/resources'
+import store from '@/store'
+
+
+describe('Breadcrumb.vue', () => {
+
+  let wrapper;
+  const breadcrumbs : BreadcrumbIF[] = [
+    {
+      text: SearchSIGNINBreadcrumb.text,
+      to: { name: RouteNames.SIGN_IN }
+    },
+    {
+      text: SearchDashboardBreadcrumb.text,
+      to: { name: RouteNames.DASHBOARD }
+    }
+  ]
+
+  wrapper = mount(BcrsBreadcrumb, {
+    props: { breadcrumbs },
+    global: {
+      provide: {
+        store: store
+      },
+      mocks: {
+        $route: {
+          name: RouteNames.DASHBOARD
+        }
+      }
+    },
+    shallow: true  // stubs out children components
+  })
+
+  it('present page is dashboard so should display breadcrumb', () => {
+    expect(wrapper.exists()).toBe(true)
```

Comment:
can you verify the props in here too please? And the current route too

Reply:
I did try now to verify most as I can